starport/pkg/plushhelpers/plushhelpers.go

```diff
@@ -4,16 +4,59 @@ import (
 	"strings"
 
 	"github.com/gobuffalo/plush"
+	"github.com/tendermint/starport/starport/templates/field"
```

Comment:
since plushhelper uses templates/, I think we cannot have this package under pkg/ anymore.

Reply:
make sense. I moved the plus helpers to `templates/field` pkg because all remain helpers is related to the fields

https://github.com/tendermint/starport/pull/1579/commits/0cb41a41f189318d6d224ba99d0d954de0e44f91